blog/PyMC_Past_Present_Future.md

```diff
@@ -0,0 +1,166 @@
+(PyMC_past_present_future)=
+# PyMC: Past, Present, and Future (PyMCon 2020)
+
+:::{post} January 25, 2023
+:tags: events, community, talks
+:category: news, testimonial
+:author: Sangam SwadiK, Rowan Schaefer
+:image: 0
+:exclude:
+
+Summary of Chris Fonnesbeck's PyMCon 2020 keynote.
+:::
+
+
+- Table of Contents
+    1. Background
+    2. PyMC Development Timeline
+        - Early 2000s
+        - PyMC 1.0
+        - PyMC 2.0
+        - PyMC 3.0
+        - Post-3.0
+    3. Applications
+    4. Present Day
+    5. Contributing
+
+At the [2020 PyMCon conference](https://pymc-devs.github.io/pymcon/about), Chris Fonnesbeck discussed the history and future of PyMC in his talk ["PyMC: Past, Present, and Future"](https://www.youtube.com/watch?v=RXODDVK7B-g&list=PLRD4U7ajWjSORb7M-VjtnvRUrfG6jXqOP&index=4). 
+
+In the talk, he discussed the broader context of probabilistic programming in the early 2000s, outlined the challenges and successes of early development, and provided insights into the future direction of the project. This blog provides a summary of the talk, which can be [viewed in full on Youtube](https://www.youtube.com/watch?v=RXODDVK7B-g&list=PLRD4U7ajWjSORb7M-VjtnvRUrfG6jXqOP&index=4), and a timeline of PyMC’s history up to 2020.
+
+## Background
+
+![Background](../_static/chris-pymcon-keynote/Background.png)
+
+### **What is PyMC?**
```

Comment:
I would not use bold font in any of the headings, they already have specific css rules to look nice (and we could modify those if we really wanted a different look, I generally prefer deferring to theme developers on this though)